test/interactions/keyInteractionTests.ts

```diff
@@ -1,302 +1,327 @@
-        assert.isTrue(aCallbackCalled, "callback for \"a\" was called when \"a\" key was pressed");
+        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aKeyCode);
+        assert.isTrue(aKeyCodeCallbackCalled, "callback for \"a\" was called when \"a\" key was pressed");
 
-        aCallbackCalled = false;
-        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aCode, { repeat : true });
-        assert.isFalse(aCallbackCalled, "callback for \"a\" was not called when keydown was fired the second time");
+        aKeyCodeCallbackCalled = false;
+        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aKeyCode, { repeat : true });
+        assert.isFalse(aKeyCodeCallbackCalled, "callback for \"a\" was not called when keydown was fired the second time");
 
-        keyInteraction.offKeyPress(aCode, aCallback);
+        keyInteraction.offKeyPress(aKeyCode, aKeyCodeCallback);
         svg.remove();
       });
 
       it("does not fire the callback if the key is pressed and held down outside, then the mouse moved inside", () => {
-        keyInteraction.onKeyPress(aCode, aCallback);
+        keyInteraction.onKeyPress(aKeyCode, aKeyCodeCallback);
         keyInteraction.attachTo(component);
 
         TestMethods.triggerFakeMouseEvent("mouseout", component.background(), -100, -100);
-        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aCode);
+        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aKeyCode);
         TestMethods.triggerFakeMouseEvent("mouseover", component.background(), 100, 100);
-        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aCode, { repeat : true });
-        assert.isFalse(aCallbackCalled, "callback for \"a\" was not called");
+        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aKeyCode, { repeat : true });
+        assert.isFalse(aKeyCodeCallbackCalled, "callback for \"a\" was not called");
 
-        keyInteraction.offKeyPress(aCode, aCallback);
+        keyInteraction.offKeyPress(aKeyCode, aKeyCodeCallback);
         svg.remove();
       });
     });
 
     describe("onKeyRelease", () => {
+      let svg: d3.Selection<void>;
+      let component: Plottable.Component;
+      let keyInteraction: Plottable.Interactions.Key;
+
+      let aKeyCode: number;
+      let aKeyCodeCallbackCalled: boolean;
+      let aKeyCodeCallback: Plottable.KeyCallback;
+
+      beforeEach(() => {
+        aKeyCode = 65;
+        component = new Plottable.Component();
+        keyInteraction = new Plottable.Interactions.Key();
+        aKeyCodeCallbackCalled = false;
+        aKeyCodeCallback = () => aKeyCodeCallbackCalled = true;
+        svg = TestMethods.generateSVG();
+        component.renderTo(svg);
+      });
+
       it("doesn't fire callback if key was released without being pressed", () => {
```

Comment:
Same as above how we should just have a base unit test about setting the `onKeyRelease` callback